ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_mathematical_functions.py

```diff
@@ -2819,6 +2819,38 @@ def test_jax_remainder(
     )
 
 
+# rint
+@handle_frontend_test(
+    fn_tree="jax.numpy.rint",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=-100,
+        max_value=100,
+        min_num_dims=1,
+        max_num_dims=3,
```

Comment:
I don't think there's a need to specify this parameter as the input can be any `n-dimensional` array